qds_sdk/commands.py

```diff
@@ -87,10 +87,24 @@ def run(cls, **kwargs):
         Returns:
             Command object
         """
+        err_pointer, tmp_pointer, new_bytes = 0, 0, 0
         cmd = cls.create(**kwargs)
         while not Command.is_done(cmd.status):
             time.sleep(Qubole.poll_interval)
             cmd = cls.find(cmd.id)
+            if kwargs.get('print_logs_live', False):
+                log, err_length, tmp_length = cmd.get_log_partial(err_pointer, tmp_pointer)
+
+                if err_length != "0":
+                    err_pointer += int(err_length)
+                    new_bytes = int(err_length) + int(tmp_length) - tmp_pointer
+                    tmp_pointer = int(tmp_length)
+                else:
+                    tmp_pointer += int(tmp_length)
+                    new_bytes = int(tmp_length)
+
+                if len(log) > 0 and new_bytes > 0:
+                    print >>sys.stderr, log[-new_bytes:]
```

Comment:
Should we write the error logs to a file (default will be `sys.stderr`)? This way we can provide flexibility to the user to save logs in any file and not just `sys.stderr`.  

Please check out the comment 1 on [this](http://stackoverflow.com/questions/5574702/how-to-print-to-stderr-in-python) stackoverflow link

As a side note, `print >> sys.stderr` is not supported in Python 3.